### ISSUE TITLE
White icon for main navigation tab

### ISSUE THREAD

**Issue by kru** [RBCA/blue-collar#95]
Hi @gthall1, I need white tab icons for the main navigation below. 
Can you hand me the link to the icon?

Thank you.

<img width="275" alt="tabbar" src="https://user-images.githubusercontent.com/11352023/39140968-7b40091e-4750-11e8-999c-305bd98194f0.png">

**Comment by gthall1** [comment]
yeah i can make white ones right quick, but where are those white?  they're blue in all the designs

**Comment by kru** [comment]
@AaronVasquez can you help me here? about the color scheme for the active tab in the google doc QA.

**Comment by kru** [comment]
Nice, I'll implement this to the tab.
Thank you @gthall1
